packages/manager/modules/sms/src/sms/dashboard/telecom-sms-dashboard.html

```diff
@@ -30,7 +25,95 @@
+                        <dd data-ng-bind="$ctrl.statistics.credits"></dd>
+                        <dd>
+                            <a
+                                class="oui-link"
+                                href="{{:: $ctrl.getReloadCreditLink() }}"
+                            >
+                                <span
+                                    data-translate="sms_statictics_credit_reload"
+                                ></span>
+                            </a>
+                        </dd>
+                    </dl>
+                </div>
+                <div class="col-md-3">
+                    <dl class="oui-description">
+                        <dt data-translate="sms_statistics"></dt>
+                        <dd>
+                            <oui-select
+                                match="label"
+                                model="$ctrl.statisticFilter"
+                                items="$ctrl.statisticsFilters"
+                                on-change="$ctrl.getStatistics()"
+                            >
+                                <span data-ng-bind="$item.label"></span>
+                            </oui-select>
+                        </dd>
+                    </dl>
+                </div>
+            </div>
+            <div class="row" data-ng-if="!$ctrl.loadingStats">
+                <div class="col-md-3 offset-md-1 oui-box oui-box_light">
+                    <dl class="oui-description">
+                        <dt
+                            data-translate="sms_statistics_delivered_number"
+                        ></dt>
+                        <dd>
+                            <strong
+                                class="oui-heading_4"
+                                data-ng-bind="$ctrl.statistics.delivered"
+                            ></strong>
+                            <span
+                                data-ng-bind="$ctrl.statistics.deliveredPercentage"
+                            ></span>
+                        </dd>
+                    </dl>
+                </div>
+                <div class="col-md-3 oui-box oui-box_light">
+                    <dl class="oui-description">
+                        <dt
+                            data-translate="sms_statistics_stop_subscribe_number"
+                        ></dt>
+                        <dd>
+                            <strong
+                                class="oui-heading_4"
+                                data-ng-bind="$ctrl.statistics.stoplisted"
+                            ></strong>
+                            <span
+                                data-ng-bind="$ctrl.statistics.stoplistedPercentage"
+                            ></span>
+                        </dd>
+                    </dl>
+                </div>
+            </div>
+        </oui-tile>
+
+        <div class="col-md-4 mb-3" ng-if="false">
```

Comment:
```suggestion
        <div class="col-md-4 mb-3">
```
or remove the entire section if it has no longer to be displayed

Reply:
yes forgot to remove the section :)